spec/expand/simple/lookup.spec

```diff
@@ -1,12 +1,31 @@
-%a
+%abc
 one
 two
 both
 
-%b:KEYS
+%def:KEYS
 CLUSTER
 TYPE
 
-%{b}:KEYS
+%{def}:KEYS
+CLUSTER
+TYPE
+
+%{a}bc
```

Comment:
yeah there is, I renamed `%a` above

`spec/expand/simple/a.yaml → spec/expand/simple/abc.yaml`